src/fancy-tools/probeTool.js

```diff
@@ -0,0 +1,167 @@
+/* eslint no-loop-func: 0 */ // --> OFF
+import external from './../externalModules.js';
+import baseAnnotationTool from './../base/baseAnnotationTool.js';
+// State
+import { getToolState } from './../stateManagement/toolState.js';
+import textStyle from '../stateManagement/textStyle.js';
+import toolColors from './../stateManagement/toolColors.js';
+// Manipulators
+import drawHandles from './../manipulators/drawHandles.js';
+// Drawing
+import { getNewContext, draw } from './../util/drawing.js';
+import drawTextBox from '../util/drawTextBox.js';
+// Utilities
+import getRGBPixels from '../util/getRGBPixels.js';
+import calculateSUV from '../util/calculateSUV.js';
+
+const cornerstone = external.cornerstone;
+
+export default class extends baseAnnotationTool {
+  constructor (name) {
+    super({
+      name: name || 'probe',
+      supportedInteractionTypes: ['mouse', 'touch']
+    });
+  }
+
+  /**
+   * Create the measurement data for this tool with the end handle activated
+   *
+   * @param {*} eventData
+   * @returns
+   */
+  createNewMeasurement (eventData) {
+    const goodEventData =
+      eventData && eventData.currentPoints && eventData.currentPoints.image;
+
+    if (!goodEventData) {
+      console.error(
+        `required eventData not supplieed to tool ${
+          this.name
+        }'s createNewMeasurement`
+      );
+
+      return;
+    }
+
+    return {
+      visible: true,
+      active: true,
+      color: undefined,
+      handles: {
+        end: {
+          x: eventData.currentPoints.image.x,
+          y: eventData.currentPoints.image.y,
+          highlight: true,
+          active: true
+        }
+      }
+    };
+  }
+
+  /**
+   *
+   *
+   * @param {*} element
+   * @param {*} data
+   * @param {*} coords
+   * @returns
+   */
+  pointNearTool (element, data, coords) {
+    const hasStartAndEndHandles =
+      data && data.handles && data.handles.start && data.handles.end;
+    const validParameters = hasStartAndEndHandles;
+
+    if (!validParameters) {
+      console.warn(
+        `invalid parameters supplieed to tool ${this.name}'s pointNearTool`
+      );
+    }
+
+    if (!validParameters || data.visible === false) {
+      return false;
+    }
+
+    const probeCoords = external.cornerstone.pixelToCanvas(element, data.handles.end);
+
+    return external.cornerstoneMath.point.distance(probeCoords, coords) < 5;
```

Comment:
I'm seeing `5` float around a lot, but I think `touch` is having a hard time with such a low number.